apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -199,7 +207,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
         currentUserMembership={props.currentUserMembership}
       />
     ),
-    limits: <EventLimitsTab eventType={eventType} />,
+    limits: <EventLimitsTab eventType={{ eventType: eventType }} />,
```

Comment:
Why?

Reply:
Since I was using `const { eventType } = props.eventType;` i had to pass it in as an object, but now that that's fixed we can use this